### PR TITLE
Prohibit mix-and-match of AMQP separators

### PR DESCRIPTION
**Pull request by jskeet**
(As discussed in yesterday's meeting.)

I've kept this brief as I don't feel it merits a huge amount of detail/motivation, but we can add more if folks think it beneficial.

### CODE REVIEW

File: cloudevents/bindings/amqp-protocol-binding.md [hunk 1]
```diff
@@ -175,7 +175,9 @@ CloudEvent attributes MUST be prefixed with either "cloudEvents_" or
 The '\_' separator character SHOULD be preferred in the interest of
 compatibility with JMS 2.0 clients and JMS message selectors where the ':'
 separator is not permitted for property identifiers (see section 3.8.1.1 of
-[JMS2.0][JMS20]).
+[JMS2.0][JMS20]). Any single message MUST use the same separator for all
+CloudEvents attributes, but single queue MAY contain messages which use
```
Comment (duglin) [inline cloudevents/bindings/amqp-protocol-binding.md:179]: s/but single/but a single/    perhaps?

Comment (jskeet) [reply cloudevents/bindings/amqp-protocol-binding.md]: Fixed, thanks.

**Comment by duglin** [comment]
LGTM

**Comment by duglin** [comment]
Approved on the 9/8 call

**Pull request merged**